src/binary/commander-options.ts

```diff
@@ -49,14 +49,16 @@ export const destConnection = new Option(
   '--dest-connection <connection>',
   'Destination connection id'
 );
-export const srcPort = new Option(
-  '--src-port <port>',
-  'Source port to create channel'
-);
-export const destPort = new Option(
-  '--dest-port <port>',
-  'Destination port to create channel'
-);
+export const srcPort = (descriptionPostfix = '') =>
+  new Option(
+    '--src-port <port>',
+    `Source port to create channel${descriptionPostfix}`
```

Comment:
What's the `descriptionPostfix` for?

Reply:
`srcPort` and `destPort` are used by more than one command, however, only one set their default values. I used `descriptionPostfix` to append that information:

```ts
  .addOption(srcPort(` (default: ${ics20Defaults.port})`))
```